.gitignore

```diff
@@ -26,3 +26,6 @@ cover.html
 
 # Local binary folder
 bin/
+
+# Mac OS files
+**/.DS_Store
```

Comment:
I think this file doesn't end on a newline character. Can we make sure it does?

Reply:
We could probably get some formatters in the linting task to automatically do that. Let me try to find a good one (or do you have anything in mind?)